core/templates/domain/opportunity/skill-opportunity.model.spec.ts

```diff
@@ -16,18 +16,14 @@
  * @fileoverview Unit tests for SkillOpportunityObjectFactory.
  */
 
-import { SkillOpportunityBackendDict, SkillOpportunityObjectFactory } from
-  'domain/opportunity/SkillOpportunityObjectFactory';
+import { SkillOpportunity, SkillOpportunityBackendDict } from
+  'domain/opportunity/skill-opportunity.model';
 
 describe('Skill opportunity object factory', () => {
```

Comment:
Do we need to change the test message here?

Reply:
done!